src/main/java/seedu/duke/Storage.java

```diff
@@ -290,11 +315,12 @@ public void loadProperty(File propertyFile) {
      *
      * @param pairFile The file that contains the pairing file.
      */
-    public void loadPair(File pairFile) {
+    public void loadPair(File pairFile) throws FileNotFoundException {
```

Comment:
Good job throwing an exception from this method to capture file loading irregularities in the method calling it. Helps with SLAP